frontend/src/config/constants.ts

```diff
@@ -0,0 +1,150 @@
+export const PATIENT_TERM = "person";
+export const PATIENT_TERM_CAP = "Person";
+export const PATIENT_TERM_PLURAL = "people";
+export const PATIENT_TERM_PLURAL_CAP = "People";
+
+export const stateCodes = [
+  "AK",
+  "AL",
+  "AR",
+  "AS",
+  "AZ",
+  "CA",
+  "CO",
+  "CT",
+  "DC",
+  "DE",
+  "FL",
+  "FM",
+  "GA",
+  "GU",
+  "HI",
+  "IA",
+  "ID",
+  "IL",
+  "IN",
+  "KS",
+  "KY",
+  "LA",
+  "MA",
+  "MD",
+  "ME",
+  "MH",
+  "MI",
+  "MN",
+  "MO",
+  "MP",
+  "MS",
+  "MT",
+  "NC",
+  "ND",
+  "NE",
+  "NH",
+  "NJ",
+  "NM",
+  "NV",
+  "NY",
+  "OH",
+  "OK",
+  "OR",
+  "PA",
+  "PR",
+  "PW",
+  "RI",
+  "SC",
+  "SD",
+  "TN",
+  "TX",
+  "UT",
+  "VA",
+  "VI",
+  "VT",
+  "WA",
+  "WI",
+  "WV",
+  "WY",
+];
+
+export const languages: Language[] = [
```

Comment:
I don't love that we have to type this full list twice. Could this be a map instead of a list? Not sure what the value would be (maybe just `true`?), but that would let you define `Language` as `keyof languages`

Reply:
The trick here is that we're defining them as _types_ in addition to constants, and we use this same pattern for Ethnicity, Gender, YesNo, and Role – it's just less offensive there because there are fewer items in the enumerations.

I'd prefer just using enums (for all of these things!), but that bucks the existing pattern. It _would_ make my separate work w/ Tribal Affiliations much less gross / repetitive.

What do you think about enums here @jeskew-gov?

Reply:
Enums could work, but do you still need to type every member twice? E.g.,

```typescript
enum Language {
  English = "English",
  Spanish = "Spanish",
  ...
}
```

 It's been a while since I used them in Typescript, but I recall that if you didn't provide a string initializer, the enum would default to serializing them as integers

Reply:
Yes, that's correct. I think another option would be to use a const array with a typeof, so something like:

```
const languages = ["English, "Spanish", ...];
type Language = typeof languages[number]
```

Reply:
Nice! I hadn't seen `const` assertions before, but I think that's exactly what we want.

Reply:
@jeskew-gov I played around with this a bit and talked offline with @TimBest: unfortunately even the const assertion option won't work for what we're trying to accomplish with the constants _and_ the type definitions together.

There's not a good way to DRY this up right now